### PR TITLE
Button: Updating stories to fix a11y issues

### LINKED ISSUE #21542: Button a11y: Issues with the use of the component in storybook
We found the following problems related to the use of the `Button` in storybook during the `Button` a11y review (#20917):
- We should remove the word "button" from the text so that it is not read twice by screen readers.
- We should add labels to all icon-only buttons.
- We should document that all the text in the `CompoundButton` is part of its name and hence should be concise.

### PR DESCRIPTION
**Pull request by khmakoto**
## PR Description

This PR fixes a number of a11y issues in our button stories:
- Removes the word "button" from the text so that it is not read twice by screen readers.
- Adds accessible labels to all icon-only buttons.
- Documents that all the text in the `CompoundButton` is part of its name and hence should be concise.

This PR also removes some `// codesandbox-dependency: @fluentui/react-button ^9.0.0-beta` comments that we missed when we made an attempt to remove them.
## Related Issue(s)

Fixes #21542

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `27.932 kB`<br />`8.025 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |   `33.196 kB`<br />`8.987 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |   `29.584 kB`<br />`8.598 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |   `35.729 kB`<br />`9.676 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |   `37.209 kB`<br />`8.627 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `48.512 kB`<br />`14.352 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `43.987 kB`<br />`13.188 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |     `7.615 kB`<br />`3.23 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |    `8.893 kB`<br />`3.675 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `7.806 kB`<br />`3.357 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `165.744 kB`<br />`46.848 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.479 kB`<br />`10.625 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/96d9c93544464bcc5ec36b7c5d724b77ae499c73'>96d9c93544464bcc5ec36b7c5d724b77ae499c73</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Avatar-mount_base.html">725</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Avatar-mount.html">651</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/BaseButton-mount_base.html">806</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/BaseButton-mount.html">796</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Breadcrumb-mount_base.html">2045</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Breadcrumb-mount.html">2059</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ButtonNext-mount_base.html">408</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ButtonNext-mount.html">396</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Checkbox-mount_base.html">1188</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Checkbox-mount.html">1326</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/CheckboxBase-mount_base.html">1140</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/CheckboxBase-mount.html">1011</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ChoiceGroup-mount_base.html">3946</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ChoiceGroup-mount.html">4022</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ComboBox-mount_base.html">854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ComboBox-mount.html">862</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/CommandBar-mount_base.html">8612</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/CommandBar-mount.html">8576</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ContextualMenu-mount_base.html">7171</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ContextualMenu-mount.html">7105</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/DefaultButton-mount_base.html">971</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/DefaultButton-mount.html">982</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/DetailsRow-mount_base.html">3120</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/DetailsRow-mount.html">3205</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/DetailsRowFast-mount_base.html">3174</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/DetailsRowFast-mount.html">3137</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/DetailsRowNoStyles-mount_base.html">3028</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/DetailsRowNoStyles-mount.html">2893</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Dialog-mount_base.html">2495</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Dialog-mount.html">2497</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/DocumentCardTitle-mount_base.html">174</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/DocumentCardTitle-mount.html">146</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Dropdown-mount_base.html">2676</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Dropdown-mount.html">2687</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FluentProviderNext-mount_base.html">1562</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FluentProviderNext-mount.html">1762</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FluentProviderWithTheme-mount_base.html">129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FluentProviderWithTheme-mount.html">136</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">81</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FluentProviderWithTheme-virtual-rerender.html">97</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">167</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">152</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FocusTrapZone-mount_base.html">1452</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FocusTrapZone-mount.html">1387</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FocusZone-mount_base.html">1569</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/FocusZone-mount.html">1588</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/IconButton-mount_base.html">1362</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/IconButton-mount.html">1404</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Label-mount_base.html">296</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Label-mount.html">283</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Layer-mount_base.html">2284</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Layer-mount.html">2532</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Link-mount_base.html">427</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Link-mount.html">425</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/MakeStyles-mount_base.html">1424</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/MakeStyles-mount.html">1414</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/MenuButton-mount_base.html">1271</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/MenuButton-mount.html">1131</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/MessageBar-mount_base.html">1697</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/MessageBar-mount.html">1754</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Nav-mount_base.html">2674</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Nav-mount.html">2661</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/OverflowSet-mount_base.html">886</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/OverflowSet-mount.html">905</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Panel-mount_base.html">2045</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Panel-mount.html">2217</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Persona-mount_base.html">659</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Persona-mount.html">674</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Pivot-mount_base.html">1163</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Pivot-mount.html">1117</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/PrimaryButton-mount_base.html">1096</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/PrimaryButton-mount.html">1118</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Rating-mount_base.html">6520</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Rating-mount.html">6108</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/SearchBox-mount_base.html">1096</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/SearchBox-mount.html">1167</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Shimmer-mount_base.html">2214</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Shimmer-mount.html">2153</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Slider-mount_base.html">1633</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Slider-mount.html">1497</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/SpinButton-mount_base.html">4111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/SpinButton-mount.html">4174</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Spinner-mount_base.html">395</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Spinner-mount.html">361</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/SplitButton-mount_base.html">2551</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/SplitButton-mount.html">2689</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Stack-mount_base.html">409</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Stack-mount.html">478</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/StackWithIntrinsicChildren-mount_base.html">1795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/StackWithIntrinsicChildren-mount.html">2003</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/StackWithTextChildren-mount_base.html">4109</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/StackWithTextChildren-mount.html">4388</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/SwatchColorPicker-mount_base.html">9379</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/SwatchColorPicker-mount.html">9567</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/TagPicker-mount_base.html">2222</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/TagPicker-mount.html">2179</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/TeachingBubble-mount_base.html">10860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/TeachingBubble-mount.html">10607</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Text-mount_base.html">341</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Text-mount.html">347</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/TextField-mount_base.html">1152</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/TextField-mount.html">1166</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ThemeProvider-mount_base.html">990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ThemeProvider-mount.html">1029</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ThemeProvider-virtual-rerender_base.html">542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ThemeProvider-virtual-rerender.html">487</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1522</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1539</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Toggle-mount_base.html">717</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/Toggle-mount.html">649</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/buttonNative-mount_base.html">137</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21679/perf-test/buttonNative-mount.html">141</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9423"/>

File: packages/react-button/src/components/MenuButton/stories/MenuButtonDisabled.stories.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const Disabled = () => {
       </Menu>
       <Menu>
         <MenuTrigger>
-          <MenuButton disabled>This is disabled</MenuButton>
+          <MenuButton disabled>Disabled</MenuButton>
```
Comment (smhigley) [inline packages/react-button/src/components/MenuButton/stories/MenuButtonDisabled.stories.tsx:23]: Maybe this could be "Disabled state" (and also Enabled state?). Just thinking that "disabled button disabled" is a little bit of a weird aural UI

```suggestion
          <MenuButton disabled>Disabled State</MenuButton>
```

File: packages/react-button/src/components/ToggleButton/stories/ToggleButtonChecked.stories.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
-import { ToggleButton } from '../../../ToggleButton'; // codesandbox-dependency: @fluentui/react-button ^9.0.0-beta
+import { ToggleButton } from '../../../ToggleButton';
 
 export const Checked = () => (
   <>
-    <ToggleButton checked={true}>A checked toggle button</ToggleButton>
+    <ToggleButton checked={true}>Checked</ToggleButton>
```
Comment (smhigley) [inline packages/react-button/src/components/ToggleButton/stories/ToggleButtonChecked.stories.tsx:6]: Similar to the "disabled" button label, I think having the label just be "checked" or "unchecked" might be a little weird b/c the state will also be announced by screen readers. Maybe something like this?:
```suggestion
    <ToggleButton checked={true}>Controlled checked state</ToggleButton>
```

File: packages/react-button/src/components/ToggleButton/stories/ToggleButtonChecked.stories.tsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
-    <ToggleButton checked={false}>An unchecked toggle button</ToggleButton>
+    <ToggleButton checked={false}>Unchecked</ToggleButton>
```
Comment (smhigley) [inline packages/react-button/src/components/ToggleButton/stories/ToggleButtonChecked.stories.tsx:8]:
```suggestion
    <ToggleButton checked={false}>Controlled unchecked state</ToggleButton>
```

File: packages/react-button/src/components/SplitButton/stories/SplitButtonAppearance.stories.tsx [hunk 4]
```diff
@@ -2,15 +2,13 @@ import * as React from 'react';
...
+import { SplitButton, MenuButtonProps } from '../../../index';
 
 export const Appearance = () => (
   <>
     <Menu positioning="below-end">
       <MenuTrigger>
-        {(triggerProps: MenuButtonProps) => (
-          <SplitButton menuButton={triggerProps} primaryActionButton={'This is a split button'} />
-        )}
+        {(triggerProps: MenuButtonProps) => <SplitButton menuButton={triggerProps} primaryActionButton="Default" />}
```
Comment (smhigley) [inline packages/react-button/src/components/SplitButton/stories/SplitButtonAppearance.stories.tsx:11]: The splitbutton menubuttons (here and in the other splitbutton stories) still don't have labels, but LMK if that's intended to be fixed in a separate PR

Comment (khmakoto) [reply packages/react-button/src/components/SplitButton/stories/SplitButtonAppearance.stories.tsx:11]: Yeah, I intend to fix those in a separate PR given that that's probably going to be more inside of the component itself to give it a default vs in the stories.

**Review by behowell — COMMENTED** [review]
Added several wording-related comments. Most of them apply to all of the button types, but I stopped adding comments after a few types to avoid sounding like a broken record.. assume I repeated all of these comments for the rest of the examples :).

File: packages/react-button/src/components/Button/stories/ButtonDefault.stories.tsx [hunk 5]
```diff
@@ -2,5 +2,5 @@ import * as React from 'react';
 import { Button, ButtonProps } from '../../../Button';
 
 export const Default = (props: ButtonProps) => {
-  return <Button {...props}>Button</Button>;
+  return <Button {...props}>Default</Button>;
```
[the remark this answers is not in the document]
Comment (khmakoto) [reply packages/react-button/src/components/Button/stories/ButtonDefault.stories.tsx:5]: We could leave it as "Example", part of this PR is removing the word "button" from the `Button` text because it sounds weird in screen readers due to repetition.

File: packages/react-button/src/components/Button/stories/ButtonWithLongText.stories.tsx [hunk 6]
```diff
@@ -3,8 +3,8 @@ import { Button } from '../../../Button';
 
 export const WithLongText = () => (
   <>
-    <Button>Text</Button>
-    <Button>Text truncates after it hits the max width token value</Button>
+    <Button>Short text</Button>
+    <Button>Long text truncates after it hits the max width token value</Button>
```
[the remark this answers is not in the document]
Comment (khmakoto) [reply packages/react-button/src/components/Button/stories/ButtonWithLongText.stories.tsx:7]: You're right, this should only say "after it hits the max width of the component". I'll make the change.

File: packages/react-button/src/components/Button/stories/ButtonIcon.stories.tsx [hunk 7]
```diff
@@ -4,11 +4,11 @@ import { Button } from '../../../Button';
...
 export const Icon = () => (
   <>
-    <Button icon={<CalendarMonthRegular />}>Text</Button>
+    <Button icon={<CalendarMonthRegular />}>With icon before contents</Button>
     <Button icon={<CalendarMonthRegular />} iconPosition="after">
-      Text
+      With icon after contents
     </Button>
-    <Button icon={<CalendarMonthRegular />} />
+    <Button icon={<CalendarMonthRegular />} aria-label="Calendar icon" />
```
[the remark this answers is not in the document]
Comment (smhigley) [reply packages/react-button/src/components/Button/stories/ButtonIcon.stories.tsx:11]: I was a bit ambivalent on whether I should recommend changing the name to include "Calendar" just for these or for all of them. The reasoning is kind of threefold:
- The heading above the examples already sets up that these are icon examples
-  Mentioning which icon is present lets someone using a screen reader communicate with a sighted person about the examples more effectively
- It sets up developers with the pattern of using `aria-label` (or a Tooltip label) as the alt text for the icon

It's not a super strong preference though, since I think the docs will be pretty reasonable either way. I also like the idea of wrapping it in a tooltip with `relationship="label"`. It could even be a hybrid, like `content="Calendar icon only"` :)

**Review by smhigley — APPROVED** [review]
LGTM no matter which approach you decide for naming the buttons with icons :)
